### PR TITLE
fix(HTTP): Missing handlers

### PR DESCRIPTION
**Pull request by kehiy**
## Description
I worte getAccountByNumber and getValidatorByNumber handler and also test for the handlers.

## Related issue(s)
feat: #528

### CODE REVIEW

**Comment by kehiy** [comment]
@b00f I think you can review this PR know!

File: www/http/account.go [hunk 1]
```diff
@@ -26,3 +29,36 @@ func (s *Server) GetAccountHandler(w http.ResponseWriter, r *http.Request) {
...
+func (s *Server) GetAccountByNumberHandler(w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
+
+	num, err := strconv.ParseInt(vars["number"], 10, 32)
+	if err != nil {
+		s.writeError(w, err)
+		return
+	}
+
+	if num > math.MaxInt32 || num < math.MinInt32 {
```
Comment (b00f) [inline www/http/account.go:43]: This check here is not necessary. HTTP is just a wrapper and should have not logic.

File: www/http/validator.go [hunk 2]
```diff
@@ -30,3 +34,41 @@ func (s *Server) GetValidatorHandler(w http.ResponseWriter, r *http.Request) {
...
+func (s *Server) GetValidatorByNumberHandler(w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
+
+	num, err := strconv.ParseInt(vars["number"], 10, 32)
+	if err != nil {
+		s.writeError(w, err)
+		return
+	}
+
+	if num > math.MaxInt32 || num < math.MinInt32 {
```
Comment (b00f) [inline www/http/validator.go:48]: This check is not necessary. HTTP is just a wrapper and should have any logic. 
Please don't forget to remove the tests for this block

**Review by b00f — APPROVED** [review]

**Comment by b00f** [comment]
@kehiy Everything looks good. Please not that HTTP is a just wrapper over gRPC. 
So, it is better to just call the gRPC, and let gRPC validates the data

**Comment by kehiy** [comment]
> @kehiy Everything looks good. Please not that HTTP is a just wrapper over gRPC. So, it is better to just call the gRPC, and let gRPC validates the data

that't mean I need just remove the checks and test for them?

File: www/http/account_test.go [hunk 3]
```diff
@@ -65,4 +66,43 @@ func TestAccount(t *testing.T) {
 		assert.Equal(t, w.Code, 400)
 		fmt.Println(w.Body)
 	})
+
+	t.Run("Shall return an account", func(t *testing.T) {
```
Comment (b00f) [inline www/http/account_test.go:70]: Let's move these tests to a new function: `TestAccountByNumber`

Also, try to cover the situation that account can't be found, like number is `acc.Number()+1`

File: www/http/validator_test.go [hunk 4]
```diff
@@ -63,4 +64,45 @@ func TestValidator(t *testing.T) {
 		assert.Equal(t, w.Code, 400)
 		fmt.Println(w.Body)
 	})
+
+	t.Run("Shall return a validator", func(t *testing.T) {
```
Comment (b00f) [inline www/http/validator_test.go:68]: Let's move these tests to a new function: `TestValidatorByNumber`

Also, try to cover the situation that validator can't be found, like number is `val.Number()+1`

**Review by b00f — APPROVED** [review]

**Comment by b00f** [comment]
> > @kehiy Everything looks good. Please not that HTTP is a just wrapper over gRPC. So, it is better to just call the gRPC, and let gRPC validates the data
> 
> that't mean I need just remove the checks and test for them?

Correct. It is OK now

**Review by b00f — APPROVED** [review]

**Review by b00f — APPROVED** [review]

**Pull request merged**